packages/one-app-bundler/bin/webpack-bundle-externals.js

```diff
@@ -0,0 +1,55 @@
+const path = require('path');
+const webpack = require('webpack');
+const readPkgUp = require('read-pkg-up');
+const chalk = require('chalk');
+const { snakeCase } = require('lodash');
+const { EXTERNAL_PREFIX } = require('..');
+
+module.exports = function bundleExternals(label) {
+  const { packageJson } = readPkgUp.sync();
+  const { version } = packageJson;
+  const { bundler } = packageJson['one-amex'];
+
+  if (!bundler.sharedExternals || bundler.sharedExternals.length === 0) {
+    return;
+  }
+
+  if (label === 'browser') {
+    bundler.sharedExternals.forEach((external) => {
+      webpack({
+        entry: path.resolve(process.cwd(), `node_modules/${external}`),
+        output: {
+          // This makes the global variable name `__holocron_external_<package-name>`
+          // eg. `__holocron_external_is_even`
```

Comment:
Note: we need to test scoped pkgs

Reply:
Note: research if we can externalize a file of pkg, for example `one-app/router`